### PR TITLE
fix docstring issues in torch.utils.tensorboard

### LINKED ISSUE #112637: Fix docstring errors in summary.py, _caffe2_graph.py, writer.py, _pytorch_graph.py, constants.py, _utils.py, _proto_graph.py
Please fix the following issues. To test locally, run: 
 ```
pydocstyle path-to-file --count
```
Run this command before and after fixing the errors. This command prints out the number of errors at the end of the output. Paste the output to your PR description with the number before and after. 
- **File**: `torch/utils/tensorboard/_proto_graph.py`, **Entity**: `attr_value_proto`, **Line**: 8, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_proto_graph.py`, **Entity**: `attr_value_proto`, **Line**: 8, **Description**: First line should end with a period (not 'g')
- **File**: `torch/utils/tensorboard/_proto_graph.py`, **Entity**: `attr_value_proto`, **Line**: 8, **Description**: First line should be in imperative mood (perhaps 'Create', not 'Creates')
- **File**: `torch/utils/tensorboard/_proto_graph.py`, **Entity**: `tensor_shape_proto`, **Line**: 23, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_proto_graph.py`, **Entity**: `tensor_shape_proto`, **Line**: 23, **Description**: First line should end with a period (not 'g')
- **File**: `torch/utils/tensorboard/_proto_graph.py`, **Entity**: `tensor_shape_proto`, **Line**: 23, **Description**: First line should be in imperative mood (perhaps 'Create', not 'Creates')
- **File**: `torch/utils/tensorboard/_proto_graph.py`, **Entity**: `node_proto`, **Line**: 38, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_proto_graph.py`, **Entity**: `node_proto`, **Line**: 38, **Description**: First line should end with a period (not 'g')
- **File**: `torch/utils/tensorboard/_proto_graph.py`, **Entity**: `node_proto`, **Line**: 38, **Description**: First line should be in imperative mood (perhaps 'Create', not 'Creates')
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `GraphPy`, **Line**: 122, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `GraphPy`, **Line**: 122, **Description**: First line should end with a period (not 'n')
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `to_proto`, **Line**: 216, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `to_proto`, **Line**: 216, **Description**: First line should end with a period (not 'd')
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `to_proto`, **Line**: 216, **Description**: First line should be in imperative mood (perhaps 'Convert', not 'Converts')
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `parse`, **Line**: 237, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `parse`, **Line**: 237, **Description**: First line should end with a period (not 's')
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `parse`, **Line**: 237, **Description**: First line should be in imperative mood; try rephrasing (found 'This')
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `graph`, **Line**: 322, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `graph`, **Line**: 322, **Description**: First line should end with a period (not 'o')
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `graph`, **Line**: 322, **Description**: First line should be in imperative mood; try rephrasing (found 'This')
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `_set_model_to_eval`, **Line**: 368, **Description**: First line should be in imperative mood; try rephrasing (found 'A')
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `_node_get`, **Line**: 385, **Description**: First line should be in imperative mood (perhaps 'Get', not 'Gets')
- **File**: `torch/utils/tensorboard/_utils.py`, **Entity**: `_prepare_video`, **Line**: 40, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_utils.py`, **Entity**: `_prepare_video`, **Line**: 40, **Description**: First line should end with a period (not ']')
- **File**: `torch/utils/tensorboard/_utils.py`, **Entity**: `_prepare_video`, **Line**: 40, **Description**: First line should be in imperative mood (perhaps 'Convert', not 'Converts')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `hparams`, **Line**: 76, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `hparams`, **Line**: 76, **Description**: First line should be in imperative mood (perhaps 'Output', not 'Outputs')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `scalar`, **Line**: 265, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `scalar`, **Line**: 265, **Description**: First line should be in imperative mood (perhaps 'Output', not 'Outputs')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `histogram_raw`, **Line**: 308, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `histogram_raw`, **Line**: 308, **Description**: First line should be in imperative mood (perhaps 'Output', not 'Outputs')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `histogram`, **Line**: 340, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `histogram`, **Line**: 340, **Description**: First line should be in imperative mood (perhaps 'Output', not 'Outputs')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `image`, **Line**: 415, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `image`, **Line**: 415, **Description**: First line should be in imperative mood (perhaps 'Output', not 'Outputs')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `image_boxes`, **Line**: 453, **Description**: First line should be in imperative mood (perhaps 'Output', not 'Outputs')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `make_image`, **Line**: 482, **Description**: First line should end with a period (not 'f')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `_get_tensor_summary`, **Line**: 734, **Description**: First line should be in imperative mood (perhaps 'Create', not 'Creates')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `_get_json_config`, **Line**: 791, **Description**: First line should be in imperative mood (perhaps 'Parse', not 'Parses')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `mesh`, **Line**: 802, **Description**: First line should be in imperative mood (perhaps 'Output', not 'Outputs')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: ``, **Line**: 1, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: ``, **Line**: 1, **Description**: Multi-line docstring closing quotes should be on a separate line
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: ``, **Line**: 1, **Description**: First line should end with a period (not 'e')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `__init__`, **Line**: 55, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `__init__`, **Line**: 55, **Description**: First line should be in imperative mood (perhaps 'Create', not 'Creates')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `get_logdir`, **Line**: 81, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_event`, **Line**: 85, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_event`, **Line**: 85, **Description**: First line should be in imperative mood (perhaps 'Add', not 'Adds')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_summary`, **Line**: 101, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_summary`, **Line**: 101, **Description**: First line should be in imperative mood (perhaps 'Add', not 'Adds')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_graph`, **Line**: 116, **Description**: First line should be in imperative mood (perhaps 'Add', not 'Adds')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_onnx_graph`, **Line**: 135, **Description**: First line should be in imperative mood (perhaps 'Add', not 'Adds')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `flush`, **Line**: 146, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `close`, **Line**: 153, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `reopen`, **Line**: 159, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `SummaryWriter`, **Line**: 168, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `SummaryWriter`, **Line**: 168, **Description**: First line should end with a period (not 'e')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `__init__`, **Line**: 187, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `__init__`, **Line**: 187, **Description**: First line should end with a period (not 's')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `__init__`, **Line**: 187, **Description**: First line should be in imperative mood (perhaps 'Create', not 'Creates')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `_check_caffe2_blob`, **Line**: 260, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `_check_caffe2_blob`, **Line**: 260, **Description**: First line should end with a period (not 'f')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `_get_file_writer`, **Line**: 275, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `get_logdir`, **Line**: 296, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_scalars`, **Line**: 394, **Description**: First line should be in imperative mood (perhaps 'Add', not 'Adds')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_histogram_raw`, **Line**: 501, **Description**: First line should be in imperative mood (perhaps 'Add', not 'Adds')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_pr_curve`, **Line**: 981, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_pr_curve`, **Line**: 981, **Description**: First line should be in imperative mood (perhaps 'Add', not 'Adds')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_pr_curve_raw`, **Line**: 1033, **Description**: First line should be in imperative mood (perhaps 'Add', not 'Adds')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_custom_scalars_multilinechart`, **Line**: 1069, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_custom_scalars_multilinechart`, **Line**: 1069, **Description**: First line should end with a period (not 't')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_custom_scalars_marginchart`, **Line**: 1088, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_custom_scalars_marginchart`, **Line**: 1088, **Description**: First line should end with a period (not 't')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_custom_scalars`, **Line**: 1106, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_custom_scalars`, **Line**: 1106, **Description**: First line should end with a period (not 'e')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_mesh`, **Line**: 1137, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_mesh`, **Line**: 1137, **Description**: First line should end with a period (not ',')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `flush`, **Line**: 1193, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_rename_tensorflow_style`, **Line**: 42, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_convert_to_ssa`, **Line**: 82, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_add_gradient_scope`, **Line**: 217, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_add_gradient_scope`, **Line**: 217, **Description**: First line should end with a period (not 'a')
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_replace_colons`, **Line**: 243, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_replace_colons`, **Line**: 243, **Description**: First line should end with a period (not 's')
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_fill_missing_operator_names`, **Line**: 267, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_add_tf_shape`, **Line**: 325, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_add_tf_shape`, **Line**: 325, **Description**: First line should end with a period (not 'f')
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_add_tf_shape`, **Line**: 325, **Description**: First line should be in imperative mood (perhaps 'Convert', not 'Converts')
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_set_tf_attr`, **Line**: 345, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_set_tf_attr`, **Line**: 345, **Description**: First line should end with a period (not ',')
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_operator_to_node`, **Line**: 390, **Description**: First line should be in imperative mood (perhaps 'Convert', not 'Converts')
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_blob_to_node`, **Line**: 479, **Description**: First line should be in imperative mood (perhaps 'Convert', not 'Converts')
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_clear_debug_info`, **Line**: 514, **Description**: First line should be in imperative mood (perhaps 'Remove', not 'Removes')
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_check_if_forward`, **Line**: 537, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_operators_to_graph_def`, **Line**: 638, **Description**: First line should be in imperative mood; try rephrasing (found 'Main')
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `model_to_graph_def`, **Line**: 762, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `model_to_graph_def`, **Line**: 762, **Description**: First line should end with a period (not 's')
- **File**: `torch/utils/hipify/constants.py`, **Entity**: ``, **Line**: 1, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/hipify/constants.py`, **Entity**: ``, **Line**: 1, **Description**: No whitespaces allowed surrounding docstring text

cc @carljparker

### PR DESCRIPTION
**Pull request by ChanBong**
Fixes #112637 

Fixed all the issues listed. 

### Error Counts

|File | Count Before | Count now|
|---- | ---- | ---- |
|`torch/utils/tensorboard/_proto_graph.py` | 9 | 0|
|`torch/utils/tensorboard/_pytorch_graph.py` | 27 | 14|
|`torch/utils/tensorboard/_utils.py` | 5 | 2|
|`torch/utils/tensorboard/summary.py` | 27 | 12|
|`torch/utils/tensorboard/writer.py` | 42 | 4|
|`torch/utils/tensorboard/_caffe2_graph.py` | 19 | 0|
|`torch/utils/hipify/constants.py` | 2 | 0|

cc @svekars

### CODE REVIEW

**Review by ezyang — APPROVED** [review]

File: torch/utils/tensorboard/_proto_graph.py [hunk 1]
```diff
@@ -5,11 +5,7 @@
 
 
 def attr_value_proto(dtype, shape, s):
-    """Creates a dict of objects matching
-    https://github.com/tensorflow/tensorboard/blob/master/tensorboard/compat/proto/attr_value.proto
-    specifically designed for a NodeDef. The values have been
-    reverse engineered from standard TensorBoard logged data.
-    """
+    """Create a dict of objects matching https://github.com/tensorflow/tensorboard/blob/master/tensorboard/compat/proto/attr_value.proto specifically designed for a NodeDef. The values have been reverse engineered from standard TensorBoard logged data."""
```
Comment (kit1980) [inline torch/utils/tensorboard/_proto_graph.py:8]: Why does this whole URL https://github.com/tensorflow/tensorboard/blob/master/tensorboard/compat/proto/attr_value.proto needs to be in the first line? Just rephase and put it later.

Comment (ChanBong) [reply torch/utils/tensorboard/_proto_graph.py]: Fixed it

**Comment by ChanBong** [comment]
@svekars This was under the docathon-h2. Maybe you have missed the label

**Pull request closed without merging**